### PR TITLE
feat(content): More Successor hails and ship names

### PR DESCRIPTION
**Pull request by Daeridanii1**
**Content**

## Summary
The primary addition in this PR is a bunch of new hails for the Successors; some for the civilian Successor government, a couple additions to the generic High Houses hails, and at least one unique hail pattern for each one of the six High Houses. 
Also included: House Kaatrij ships now have a 50% chance to have unique names of similar form to the name of their eminent spokessquid Saajret, of Somber Sejra. Because these include English words, these are applied in the event associated with getting the translation device rather than at the start of the game (Kaatrij uses 100% standard High Houses ship naming before this).
Finally, the additions to hails required changing a few of the phrase word lists in `successors.txt`, so those have been altered or added to as appropriate.

## Testing Done
I've flown around a ton and haven't seen any missing spaces, etc, that I haven't caught. I've checked what I qualitatively thought to be the longest name combinations for Kaatrij and they don't overflow.

## Save File
This save file can be used to test these changes:
[Icy Precipitation.txt](https://github.com/user-attachments/files/17404634/Icy.Precipitation.txt)

### CODE REVIEW

File: data/successors/successor hails.txt [hunk 1]
```diff
@@ -328,39 +933,216 @@ phrase "new houses friendly hail" # Boast hails
...
+	word
+		"pride"
+	word
+		", / "
+	word
+		"the Houses, New"
+		"our noble House"
+		"grand Seineq's House"
+		"the Lum'nary party"
+		"the Twice-Wrapped Ring"
```
Comment (TheGiraffe3) [inline data/successors/successor hails.txt:1047]:
```suggestion
		"the Twice-Wrapped Ring"
	word
		" "
```

File: data/successors/successor hails.txt [hunk 2]
```diff
@@ -57,19 +59,188 @@ phrase "successor friendly hail" # Brag hails 1
...
+		" "
+	word
+		"enjoyed"
+		"seen"
+		"experienced"
+		"taken'n"
+	word
+		" "
+	phrase
+		"successor media name"
```
[the remark this answers is not in the document]
Comment (Daeridanii1) [reply data/successors/successor hails.txt:145]: `shakespeare play` is contained within `successor media name`, so I don't think this is necessary.

File: data/successors/successors.txt [hunk 3]
```diff
@@ -368,15 +482,57 @@ phrase "successor commodities (2-syllable)"
...
+		"The Last Days of House Sejra"
+		"Void Furies"
+		"We Are Ash"
+
+phrase "shakespeare play"
+	word
+		"All's Well That Ends Well"
+		"As You Like It"
+		"Hamlet"
+		"Love's Labour Lost"
```
[the remark this answers is not in the document]
Comment (TheGiraffe3) [reply data/successors/successors.txt:526]: Ah, right. In that case, you'll want to add it to `.codespell.exclude`. I assume you know how to do that?

File: data/successors/successors.txt [hunk 4]
```diff
@@ -11,9 +11,11 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
-phrase "successor nouns"
+phrase "successor noun"
```
Comment (TheGiraffe3) [inline data/successors/successors.txt:14]: Would you be willing to alphabetize these?

File: .codespell.exclude [hunk 5]
```diff
@@ -105,3 +105,4 @@ system Currus
 		"i' amazin"          # is amazing (mispronounced)
             * Care Package 2a has been replaced with a new mission, FW Deep Memorial, a set of missions and events that offers regardless of the player's choices, but which has a branch if Brower dies during the attack.
 			`	He smiles sadly. "Skaldgar always had a nose for good folks. Anyways, he told me he had a nephew named Helm who worked at the Norn spaceport. Find 'im and he can get'cha and this stone where it needs to go." The young man then looks you in the eye and gives a slow respectful nod, "For Ol' Skaldgar, may he be findin' ore in hel."`
+		"Love's Labour Lost"	#original English spelling
```
Comment (TheGiraffe3) [inline .codespell.exclude:108]:
```suggestion
		"Love's Labour Lost" # This is the original English spelling.

```

File: data/successors/successor hails.txt [hunk 6]
```diff
@@ -189,6 +442,215 @@ phrase "high houses friendly hail"
...
+		" "
+	word
+		"and blanch in fear"
+		"before our might"
+		"as we arrive"
+	word
+		". / "
+		"! / "
+	word
+		"Your House is here"
```
Comment (TheGiraffe3) [inline data/successors/successor hails.txt:525]:
```suggestion
		"Your House is here"
		"Your House has arrived"
```

File: data/successors/successor hails.txt [hunk 7]
```diff
@@ -189,6 +442,215 @@ phrase "high houses friendly hail"
...
+		"upon our people"
+	word
+		". / "
+		"! / "
+	word
+		"All knaves shall meet their bitter end"
+		"Our wrath stands equal to our love"
+		"Our might exceeds the basest foe"
+		"Our duty 'mplores us to defend"
+		"We shan't permit a violent foe"
```
Comment (TheGiraffe3) [inline data/successors/successor hails.txt:567]: To do what?

File: .codespell.exclude [hunk 8]
```diff
@@ -105,3 +105,4 @@ system Currus
 		"i' amazin"          # is amazing (mispronounced)
             * Care Package 2a has been replaced with a new mission, FW Deep Memorial, a set of missions and events that offers regardless of the player's choices, but which has a branch if Brower dies during the attack.
 			`	He smiles sadly. "Skaldgar always had a nose for good folks. Anyways, he told me he had a nephew named Helm who worked at the Norn spaceport. Find 'im and he can get'cha and this stone where it needs to go." The young man then looks you in the eye and gives a slow respectful nod, "For Ol' Skaldgar, may he be findin' ore in hel."`
+		"Love's Labour Lost"	# The standard English spelling of the play's name.
```
Comment (bene-dictator) [inline .codespell.exclude:108]:
```suggestion
		"Love's Labour Lost"
```
The checks will continue to fail unless the line is exactly the same as the one you want to exclude.

**Review by roadrunner56 — APPROVED** [review]

**Pull request merged**